screengrab/screengrab-lib/src/main/java/tools.fastlane.screengrab/locale/LocaleUtil.java

```diff
@@ -25,7 +25,8 @@ public static void changeDeviceLocaleTo(Locale locale) {
             methodGetDefault.setAccessible(true);
             Object activityManagerNative = methodGetDefault.invoke(amnClass);
 
-            Method methodGetConfiguration = amnClass.getMethod("getConfiguration");
+            Class iam = Class.forName(activityManagerNative.getClass().toString());
```

Comment:
Thanks for the heads up. The lowest I tested was api 23. I can add a work around for that.